pkgs/development/python-modules/benchopt/default.nix

```diff
@@ -0,0 +1,94 @@
+{
+  buildPythonPackage,
+  fetchPypi,
+  lib,
+  pytestCheckHook,
```

Comment:
Please move this in its own category , it should not be there, usually it is at the end of the list.